tutorial/practicals/LFRic/building_code/4_psydata/README.md

```diff
@@ -6,8 +6,10 @@ Initially we will be using kernel extraction as an example
 but a list of other PSyData applications is provided at the end.
 The ``solutions`` directory contains working versions of the
 transformation scripts and a Makefile. If you are having problems,
-you can just copy the corresponding Makefile and python script
-into this directory.
+you can look at the scripts and various makefiles in that directory.
```

Comment:
I just re-read the README - and (after your previous observation about the missing quotes :) )  I consistently use 'makefile' when I talk about makefiles in general ("There is a set of makefiles provided ..."), but capitalised with quotes when mentioning the name of a specific makefile (since the makefile filenames are capitalised). I admit I have no idea if this is correct (I tried googling, but got lost in too many makefiles doing spell checking 😆 ). Is that ok?